### ISSUE TITLE
Improve Bound-with display and request links

### ISSUE THREAD

**Issue by marlo-longley** [sul-dlss/SearchWorks#3200]
We got to parity with the current SW display with the PRs linked below. 
https://github.com/sul-dlss/SearchWorks/issues/3128
https://github.com/sul-dlss/searchworks_traject_indexer/issues/848

We also took notes in #3099 but are closing in favor of this more specific ticket.

The UI remains the same. Here are some notes on basic improvements in 2 areas of the SearchWorks UI.

![Screenshot 2023-06-02 at 16 27 40](https://github.com/sul-dlss/SearchWorks/assets/1328900/c1ae58e7-3030-49f9-88e4-4843e3b8a762)
1. The request button for SEE-OTHER bound with children. For FOLIO parity, we are creating "fake" SirsiHoldings with SEE-OTHER data. However in FOLIO this distinction doesn't exist. It's the SEE-OTHER note that is the trigger for the absense of a request button in a bound-with child, and causes this yellow-highlighted box with a link to the parent to be displayed. The SEE-OTHER note is applied when a bound-with child is located in SAL3. An improved UI would be, for all bound-with children, regardless of their location in SAL3, to display a request button in this place instead.

     To achieve this, we need the request button to link to a request for the parent item. We now have most of the needed data indexed from FOLIO to put together the correct link. However, we need to somehow communicate to Requests that this is not a typical request for the parent item, but rather a request for one of its bound-with children. We need to figure out a good data connection (URL param?) for making this clear to Requests. Then, we need to display this information in Requests so that the user understands that, when they go to the counter to pick up their item, it will have the parent's title, etc.

2. The note is something that users find confusing / too much data. My understanding is that the last part of the note (item id) is something currently being used for the FOLIO migration and won't be present afterwards. But the text that reads `Copy 1 bound with v. 5, no. 1. 5488000 (parent record's ckey).` should be improved. It would be more helpful for the user if `5488000` was replaced with the title of the parent (keeping the link). 

To summarize, some basic first steps for improving the UI would be:
- Render a request button for all bound-with children that request their parents. Send extra data to Requests that this is a bound-with child request.
- Display new information to the user in Requests about the nature of this bound-with item (this would be a ticket/task in the sul-requests repo)
- Remove overwhelming identifiers in the note content in SearchWorks and replace with the parent item's title

_Originally posted by @marlo-longley in https://github.com/sul-dlss/SearchWorks/issues/3099#issuecomment-1574444036_

**Comment by saseestone** [comment]
After the folio launch, there was some discussion in the team-taco-dance channel Course Reserves, and that a Reserve record can't be placed on a Bound with child record because it lacks an item. 

Reserves staff will need to locate the parent record, and associate the reserve record with the parent item. Chris wondered if we should be displaying the reserve information panel on both the parent and child record in SearchWorks.  That makes sense to me.

**Comment by marlo-longley** [comment]
Breaking the above comment about Course reserves out into its own ticket.

**Comment by cbeer** [comment]
We should also use the circ rules to determine the bound with parent circulation policies, e.g. https://searchworks.stanford.edu/view/10498146 should display as "In-library use" because the parent doesn't circulate. That means we probably need to pull back more item-level information for the parent.

**Comment by saseestone** [comment]
Jeanette in Metadata found this gem of a Boundwith record: https://searchworks.stanford.edu/view/1598384. It's got two copies. The first is a boundwith. The second copy is not.  I'll create another "bug" ticket because it's an issue that the second copy can't be requested rignt now, but figured this record would be useful as we think about improvements to display & requests for Boundwiths.

**Comment by dbranchini** [comment]
Screens have been designed for displaying bound-with parent and child records on the item detail view in SearchWorks. 

Bound-with - 3 child records with 3 different parents:
https://www.figma.com/proto/W3GIkPyGLeO5xMojmkZuTX/SearchWorks?page-id=453%3A2788&type=design&node-id=453-4007&viewport=2247%2C883%2C0.5&t=QreQwn6LHRwmydST-9&scaling=min-zoom&starting-point-node-id=453%3A4007&show-proto-sidebar=1

Bound-with - 1 child record **_and_** click on the parent record, "The gases of swamp rice soils," to see a combination of child and parent records:
https://www.figma.com/proto/W3GIkPyGLeO5xMojmkZuTX/SearchWorks?page-id=453%3A2788&type=design&node-id=453-4106&viewport=2247%2C883%2C0.5&t=QreQwn6LHRwmydST-9&scaling=min-zoom&starting-point-node-id=453%3A4106&show-proto-sidebar=1

Screenshots follow with a few annotations:

From first prototype:
<img width="380" alt="Screenshot 2024-02-29 at 9 31 37 AM" src="https://github.com/sul-dlss/SearchWorks/assets/6816624/068d76a3-bf1c-4704-8ed6-28db11531de7">

From second prototype:
<img width="375" alt="Screenshot 2024-02-29 at 9 31 04 AM" src="https://github.com/sul-dlss/SearchWorks/assets/6816624/d0c7b2b9-efd3-4b3d-bac9-78e4e8876a02">

<img width="384" alt="Screenshot 2024-02-29 at 9 31 18 AM" src="https://github.com/sul-dlss/SearchWorks/assets/6816624/9eb2114b-57a2-48d6-94c9-baaaedd9691b">

<img width="860" alt="Screenshot 2024-02-29 at 9 31 28 AM" src="https://github.com/sul-dlss/SearchWorks/assets/6816624/09404562-98c8-4c2c-afb9-ca26a6b4db2f">

**Comment by dbranchini** [comment]
Design specifications:

- Display of both child and parent for the "At the library" cards, should come from FOLIO data and not the 590 field. The 590 field should continue to be displayed in the **Bibliographic information > Notes**.
- Parent records follow this pattern:
Bound with:
[Child title] - linked to record detail page
[Child effective call number]
- Child records follow this pattern:
Bound with and shelved by:
[Parent title] - linked to record detail page
[Parent effective call number]
- If there are both parent and child records with the same title and record view, then they are grouped by the parent record. 
- If there are several children
-- Order by call number (sequential - I believe there's other work happening on this where it's descending if it's a date and ascending if it's a number)
-- Show one child record, with a button or link to open a modal with the rest of the children listed in a table view.
-- If there is a record with the same title, display that first, otherwise, the display the first child. 
- **NOTE** use existing styles in most cases, we are not restyling the record view. If there's already a modal or table in our library, use that. Also, it wasn't intentional to reverse or change the truck icon next to available, I'm just using the font awesome truck and the attribute fa-flip isn't an option in Figma. I'm sure there are other cases of that. However, there are few purposeful design choices that I'm hoping to implement:
-- Grey-ish background color for the bound-with details is Accent Fog Light or `#F4F4F4` from the Stanford Identity Guide.
-- "Bound with" or "Bound with and shelved by" headings are Primary Cardinal Red or `#8C1515` from the Stanford Identity Guide, 14 pixels (base size) and medium weight or 500.

**Comment by dbranchini** [comment]
IMPORTANT - I'm not sure if this will be handled by this ticket or one of the other bound with tickets, but note the following:
If a request is placed on a child item, the request is automatically routed to the parent record. 
If a request is placed on a child item that's associated with a specific volume/copy on the parent record, the request is automatically routed to that volume/copy (and not the item selector view). 
If a request is placed on a parent item, the request is routed to the parent record. There is no change here, except that additional child details are displayed in the request app.

**Comment by jcoyne** [comment]
Regarding
> Remove overwhelming identifiers in the note content in SearchWorks and replace with the parent item's title

This data is from the MARC 590 field. We aren't constructing it in Searchworks:
<img width="634" alt="Screenshot 2024-02-29 at 2 33 04 PM" src="https://github.com/sul-dlss/SearchWorks/assets/92044/120d667a-8adc-4f41-86d9-1492d6aee824">

**Comment by jcoyne** [comment]
@dbranchini so just implement the design specs and ignore what we have in the ticket description?

**Comment by dbranchini** [comment]
Yes, I started with the ticket description (really helpful) as inspiration and then designed from there. My design work has been reviewed by both Sarah and Chris.

**Comment by saseestone** [comment]
From the storytime discussion, we decided we needed an example of a Green stacks item with the parent checked out. 

I've checked out this parent record in folio-test: [a6290236](https://folio-test.stanford.edu/inventory/view/e86d3e2f-25e6-5635-8276-c944c68746fb/e557d96b-ce1f-51a2-a09f-7ff3a427ec7d/5502e87e-22cf-5f59-8cde-6edcca71eaf3?query=a6290236&sort=title)
Child record associated is: [a6290245](https://folio-test.stanford.edu/inventory/view/563e2a6d-8a30-5378-af7f-bf3067ab0a5d/88f0210b-1fdc-5da5-8572-deb4d3afe63c?query=a6290245&sort=title)

**Comment by dbranchini** [comment]
Split into three tickets - #4074 , #4075 , and #4077 and closing this ticket as the work is covered in the three separate tickets.
